### ISSUE TITLE
npm run site error

### ISSUE THREAD

**Issue by afc163** [ant-design/ant-design-pro#597]
http://github.com/ant-design/ant-design-pro

```
Failed to compile.


./src/utils/request.js
  Line 16:  Unexpected dangling '_' in '_req'  no-underscore-dangle
  Line 22:  Unexpected dangling '_' in '_res'  no-underscore-dangle
```

**Comment by afc163** [comment]
和 #7 差不多是同一类问题。

**Comment by afc163** [comment]
https://travis-ci.org/ant-design/ant-design-pro/jobs/325996325

**Comment by nikogu** [comment]
fixed: roadhog-api-doc@0.3.4
